src/zepben/auth/authenticator.py

```diff
@@ -89,7 +94,7 @@ def fetch_token(self) -> str:
             # Stored token has expired, try to refresh
             self._access_token = None
             if self._refresh_token:
-                self._fetch_token_auth0(use_refresh=True)
+                self._fetch_token_auth0()
```

Comment:
This exact call is repeated when the refresh token fails on line 102, like me recompiling code that didn't compile the first time